### DISCUSSION TITLE
STEP800 timeline

### DISCUSSION THREAD

**Discussion by kanta** [ponoor/step-series-support#3] [Announcements]
STEP400の兄弟機として、小型ステッピングモータ用を8軸制御できる[STEP800](https://ponoor.com/products/step800/)を開発しています。
すでに基板製造は開始しているのですが、コア部品であるL6470の入手に時間がかかっているため、部品実装に進めない状態です。サプライヤからの部品納品予定は7月上旬ころを見込んでいまして、予定通り進めば8月上旬には販売開始できる予定です。
アップデートがありましたら随時Twitter等でお知らせするほか、こちらのdiscussionにもポストしていきます。

As a sibling of STEP400, we are developing [STEP800](https://ponoor.com/en/products/step800/), which can control 8 small stepping motors.
We have already started manufacturing the circuit boards, but it is taking a long time to obtain the core component, L6470, so we are unable to proceed with component mounting. We expect to receive the components from our suppliers in early July, and if everything goes as planned, we will be able to start selling the product in early August.

![image](https://user-images.githubusercontent.com/1097902/118651689-c811b500-b820-11eb-8b5e-6238faa35e8d.png)
Testing a prototype

**Comment by dappertutto** [comment]
fantastic! Could it handle 8 nema 17's at once?

**Reply by kanta** [reply]
In most cases, yes! 
If the motor has a phase current of up to about 2A, it should be able to be driven with STEP 800. Most NEMA 17 motors are in this range.
In the following video, the eight yellow rings on the left side were driven by a single STEP800.
https://www.youtube.com/watch?v=Ydo_slg8oK0

**Comment by kanta** [comment]
The production version of STEP800 has finally arrived. We're almost ready to start selling it!
![image](https://user-images.githubusercontent.com/1097902/130907835-87d10e31-8570-4aaf-9957-564c71d164b7.png)

**Comment by kanta** [comment]
I have forgotten to mention about it here but STEP800 are available at [ponoor.com](https://ponoor.com/en/shop/)!
